### ISSUE TITLE
please support typescript

### ISSUE THREAD

**Issue by limit0x** [Connect2IC/connect2ic#1]
now, the code can't use in typescript, please support typescript

**Comment by MioQuispe** [comment]
> now, the code can't use in typescript, please support typescript

Yes. Typescript support is definitely coming. I'm still working on the project so consider this a pre-alpha at the moment...
